client/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as S from './styles'
+
+import { AvatarProps } from './types'
+import { lorelei } from '@dicebear/collection'
+import { createAvatar } from '@dicebear/core'
+import { useMemo } from 'react'
+
+const Avatar = ({
+  size = 'md',
+  isRound = false,
+  src = '',
+  status = '',
+}: AvatarProps) => {
+  src = useMemo(() => {
+    return createAvatar(lorelei, {
+      size: 80,
+    }).toDataUriSync()
+  }, [])
+  return (
+    <>
+      <S.Wrapper size={size} isRound={isRound} src="">
```

Comment:
```suggestion
      <S.Wrapper 
         size={size} 
         isRound={isRound} 
         src=""
      >
```
Porque o src está vazio? Ele está sendo usado?